### DISCUSSION TITLE
What is an example of a MyST-NB example that is not a Jupyterbook

### DISCUSSION THREAD

**Discussion by ketozhang** [executablebooks/meta#859] [Q&A] [answered]
It's not very clear the difference between MyST-NB and Jupyterbook. I understand the former is the core of the latter.

Unfortunately the frontpage currently points to [this gallery](https://executablebooks.org/en/latest/gallery.html) which doesn't distinguish the two either.

**Comment by NickleDave** [comment, marked as answer]
You can use MyST-NB even if you're not using jupyter book.

For example, if you have .md files in your documentation that you want to execute as if they were jupyter notebooks (even if you're not writing something that is very book-like).

Here is an example:  
https://raw.githubusercontent.com/vocalpy/crowsetta/main/doc/scripts/raven-chronister-et-al-2021.md

I use `jupytext` so I can edit and execute this file as a Jupyter notebook when I'm developing, but keep it in the .md format.  
Then when I build I use `myst-nb` instead of `myst_parser`  

Does that help?
Do you think the documentation should state that explicitly?  
I'm just a user, not a dev, but just curious about what you think would help.

I kind of think of it as the `myst` version of [`nbsphinx`](https://nbsphinx.readthedocs.io/en/0.8.9/) -- some maintainer can tell me if that's a terrible analogy though

**Reply by ketozhang** [reply]
Gotcha, so it's just `ipynb | md ` to sphinx output (HTML usually). It's not a book framework 

I think the documentation are ambiguous and the wording in the home page cause confusion:

> A Sphinx and Docutils extension for compiling Jupyter Notebooks into high quality documentation formats.

I read this to mean I can convert `.ipynb` to HTML.  Okay, that's possible, but doesn't mention Jupyter cells can also be written with markdown.
